ide/tests/build.properties

```diff
@@ -1,7 +1,6 @@
 source.. = src/test/java/
 output.. = bin/test
-bin.includes = META-INF/,\
-               .,\
+bin.includes = .,\
```

Comment:
Is this change important for this PR?  If not could we undo it?

Reply:
All of your changes sound good to me.  I am not sure how the top-level java dir got there, rather than cast/java, but it was not deliberate.  

I think the change to `ide/tests/build.properties` made a warning go away in Eclipse, which I still use, but it is not vital.

reverting the .prefs files and `TestSimpleCallGraphShape` is all fine.

Reply:
Thanks, @juliandolby.  Please let me know once you've had time to make all these changes (no hurry on my end), and then after that I'll take another look.  I expect we'll be able to land the PR soon after.